capture_packets.py

```diff
@@ -42,25 +48,22 @@ def main():
 def pull_and_validate_addrs():
   print("Pulling and validating MAC addresses")
 
-  with open(config_file_path, "r") as config_file:
-    config_json = json.load(config_file)
-
   # Throw an error on a bad MAC address or add it to the global MAC address storage
   # TODO: The validation likely won't go in this script, but we'll keep it here for now
   global mac_addrs
-  for addr in config_json["mac_addrs"]:
+  for addr in run_config.mac_addrs:
     if not re.match("[0-9a-f]{2}([:])[0-9a-f]{2}(\\1[0-9a-f]{2}){4}$", addr.lower()):
       sys.exit("Provided address " + addr + " is not a vaid MAC address.")
```

Comment:
We should avoid using sys.exit() at any point in the code. The service will be reloaded and restarted if it stops, ends, crashes for any reason so this will just cause it to go into a loop.

Reply:
This line won't survive anyways, there's a TODO above the loop saying that the validation will happen when the addrs are inputted by the user, this was so I didn't mess up with the config file lol. I could make it a log instead tho, just for sanity purposes.

Reply:
I see. And I would recommend eventually using logging.